src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
The comment above this says it was done to avoid a `.stopPropagation()` being in the simulated event and affecting the donor event. There still seem to be places in compat that would be broken by this, for example https://github.com/jquery/jquery/blob/compat/src/event.js#L897-903 when the donor event is `click` and the simulated event is `change`, with a change handler calling `event.stopPropagation()` actually stopping propagation on the original click event.